ibm_cloud_sdk_core/authenticators/basic_authenticator.py

```diff
@@ -17,23 +17,37 @@
 from .authenticator import Authenticator
 from ..utils import has_bad_first_or_last_char
 import base64
+import requests
```

Comment:
How about importing what you need directly?

Reply:
👍